src/backtrace/mod.rs

```diff
@@ -130,7 +130,7 @@ impl Outcome {
             Outcome::StackOverflow => log::error!("the program has overflowed its stack"),
             Outcome::HardFault => log::error!("the program panicked"),
             Outcome::Ok => log::info!("device halted without error"),
-            Outcome::CtrlC => log::info!("device halted by user"),
+            Outcome::CtrlC => log::info!("interrupted by user"),
```

Comment:
```suggestion
            Outcome::CtrlC => log::info!("device interrupted by user"),
```

Reply:
I changed it to the current message because it's the *probe* that was interrupted, not the device -- the device keeps running if you do `--no-reset`.

Reply:
Maybe we can actually have two different messages then?

```suggestion
            Outcome::CtrlC if opts.no_reset == true => log::info!("interrupted by user; program keeps running"),
            Outcome::CtrlC if opts.no_reset == false => log::info!("device halted by by user"),
```

Reply:
It's not, but we can just add it as an argument:

```rust
fn log(&self, no_reset: bool)
```